website/docs/docs/dbt-cloud/dbt-cloud-api/service-tokens.md

```diff
@@ -1,38 +1,90 @@
 ---
-title: "Service Account Tokens"
+title: "Service account tokens"
 id: "service-tokens"
+description: "Service account tokens help you define permissions for securing access to your dbt Cloud account and its projects."
 ---
 
-## Overview
-Service Account Admin API tokens are like [User API tokens](user-tokens), but they belong to an
-_account_, rather than _a user_.
+## About service tokens
 
-As such, they may be suitable for system-level integrations that do not run on behalf of any one user. Service Account Admin tokens may only be created or modified by users with Account Admin (Enterprise plan) or Owner (Team plan) permissions on an account. Service Account tokens may be revoked if they become lost or compromised.
+Service account tokens enable you to securely authenticate with the dbt Cloud API by assigning each token a narrow set of permissions that more precisely manages access to the API. While similar to [User API tokens](user-tokens), service account tokens belong to an account rather than a user.
 
-### Service Account Admin API tokens
+You can use service account tokens for system-level integrations that do not run on behalf of any one user. Assign any permissions set available in dbt Cloud to your service account token.
 
-<Changelog>
+:::info Enterprise and Team plan differences
+Enterprise plans can apply any permission sets available to service tokens. Team plans can apply the following permissions sets to service tokens: Account Admin, Member, Job Admin, Read-Only, and Metadata Only.
```

Comment:
```suggestion
Enterprise plans can apply any permission sets available to service tokens. Team plans can apply the following permissions sets to service tokens: Owner, Member, Job Admin, Read-Only, and Metadata Only.
```

Reply:
`Account Admin` was correct here! That's how we're displaying the permission in the UI, even for tokens on Team plans, as before these changes we referred to them as Account Admin tokens on all plans

Reply:
Fixed back to "Account Admin"  in https://github.com/dbt-labs/docs.getdbt.com/pull/1045/commits/ef7f3e46e6ecad73b856227a17feac89d892091f